x-pack/libbeat/common/cloudfoundry/cache.go

```diff
@@ -63,9 +63,9 @@ type appResponse struct {
 
 func (r *appResponse) fromStructs(app cfclient.App, err error) {
 	if err != nil {
-		cause := errors.Cause(err)
```

Comment:
@efd6 it appears we need to update go-cfclient to get this to work, because the package uses errors.Cause() under the hood. It seems you fixed this problem yourself (https://github.com/cloudfoundry-community/go-cfclient/commit/0ada4e9452d702a78a40974a95191f0ee8ab16c6) but I don't know this code at all and I don't know what the consequences of upgrading this package is, since it mentions there can be breaking changes introduced at any time.

Reply:
If you want to avoid the package upgrade then you can implement a cause function like I have there that will accept both the stdlib and pkg/errors wrappings. This is probably the safest way. I don't recall exactly why I sent that PR, but it was probably related to clean-ups here. That the version has not been bumped says that I worked around the issue in our code. This would likely have been in filebeat, or I may have just left it as being out of scope (I think the latter is more likely).

Reply:
Thanks, I just explicitly handled both forms of error wrapping to future proof it, and disabled the lint warnings for pkg/errors in this file for now.